src/main/java/org/apache/mesos/scheduler/DefaultScheduler.java

```diff
@@ -43,29 +43,45 @@
     private static final Integer PERMANENT_FAILURE_DELAY_SEC = 20 * 60;
     private static final Integer AWAIT_TERMINATION_TIMEOUT_MS = 10000;
     private final Logger logger = LoggerFactory.getLogger(getClass());
-    private final ServiceSpecification serviceSpecification;
     private final String zkConnectionString;
     private final ExecutorService executor = Executors.newFixedThreadPool(1);
     private final BlockingQueue<Collection<Object>> resourcesQueue;
+    private final String name;
```

Comment:
Style nit: What does this `name` represent ?  Service name ? Can we please rename this ?